### PR TITLE
Fix typo in data-management-tutorial.md

### PR DESCRIPTION
**Pull request by EshaMaharishi**
Looks like a typo: "With data capture enabled, you can now configure data capture" -> "With data management enabled, you can now configure data capture"

### CODE REVIEW

**Review by sguequierre — APPROVED** [review]

File: docs/tutorials/data-management-tutorial.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ For more detailed information see [Add the Data Management Service](../../servic
 
 ## Configure data capture for a component
 
-With data capture enabled, you can now configure data capture for specific components on your robot running Viam.
+With data management enabled, you can now configure data capture for specific components on your robot running Viam.
```
Comment (npentrel) [inline docs/tutorials/data-management-tutorial.md:73]:
```suggestion
With the data management service added, you can now configure data capture for specific components on your robot running Viam.
```

**Review by npentrel — APPROVED** [review]

**Comment by npentrel** [comment]
Thank you Esha!

**Comment by EshaMaharishi** [comment]
Thank you both for the speedy review!

**Comment by npentrel** [comment]
(you may have the honour of merging)

**Pull request merged**